src/zzz_od/application/hollow_zero/lost_void/lost_void_app.py

```
@@ -170,13 +181,216 @@ def check_bounty_commission_before(self) -> OperationRoundResult:
+            rect=area.rect,
+        )
+
+        # 先点击UP代理人
+        clicked = False
+        for ocr_text in ocr_result_list:
+            if 'up' in ocr_text.data.lower() and ocr_text.center.x < self.ctx.controller.standard_width // 2:
+                self.ctx.controller.click(ocr_text.center)
+                clicked = True
+                break
+
+        # 找不到UP，点击第一个
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)